src/v3/package.json

```diff
@@ -30,7 +30,10 @@
     "test:dev": "NODE_ENV=development jest",
     "test:e2e": "testcafe test/e2e/tests/**/*.js --config-file test/e2e/.testcaferc.js",
     "test:e2e:screenshots": "yarn test:e2e -s path=test/e2e/screenshots,thumbnails=false",
-    "test:report": "PREACT_APP_USE_PKCE=false NODE_ENV=test jest --coverage"
+    "test:report": "PREACT_APP_USE_PKCE=false NODE_ENV=test jest --coverage",
+    "test:vrt": "testcafe \"chrome:headless --force-device-scale-factor=1\" --config-file test/vrt/.testcaferc.js",
+    "test:vrt-ci": "VRT_CI=true yarn test:vrt",
```

Comment:
Oh I was thinking the env variable change would be set in the shell script like `export VRT_CI=true` so this script wouldn't be needed

Reply:
ohh okay ill try that

Reply:
yep looks like that works fine 👍